### ISSUE TITLE
[Hero]Blacksmith

### ISSUE THREAD

**Issue by Wujekklawy** [AngelArenaAllstars/suggestions#359]
melee, strength.

Q **[Weapon exploit]** [Pierce magic immunity on allays] change target damage type from physical to pure for x seconds. After buff ends target is disarmed. Disarm is unavoidable and not dispelable, pierce magic immunity.

W **[Guard]** [allay's only] target takes reduced damage taken, damage soaked by guard is transferred to blacksmith (can target only one hero, can be turned of at any time) Pierce spell immunity.
>work similar to blade mail but instead attacker take damage Blacksmith does. (around 40% damage taken is transferred to blacksmith, before reductions) end when blacksmith dies / get out of range / turned off.

E **[Reinforce]** [passive] increase armor and crowd control reduction. AoE.

R **[Equipment Defect]** Disarms and reduce armor of enemy in area. AoE. ground target only.

**Tree**

- **lvl 10** +hp / x% max hp heal out of combat
- **lvl 15** x% chance to block y damage / weapon exploit now apply bleeding 
>.../ stackable damage over time, when attacking with that buff
- **lvl 20** increase range of reinforce / Guard now purge debuffs 
>... / when casted
- **lvl 25** increase duration of weapon exploit / guarded target now not take damage
> buff duration on allays, disarm duration on enemies / percent of damage transfered stays same as before

(i can write Lore and flavored spell description)

**Comment by Wujekklawy** [comment]
#393

**Comment by KaoticKanine** [comment]
I take it the W is a strict support ability that has Blacksmith absorb most of the damage his target ally sustains? I guess that's obvious but it doesn't specify an ally, so I have to ask lol 

That said, really interesting hero idea :D This guy makes for a very effective counter against most Agi carries while buffing his own, with some risk thanks to the Q making damage pure and W potentially getting him killed

**Comment by Wujekklawy** [comment]
**[Edit]** 
- Refrazed a little Guard to be more accurate.
- Corrected some typos.
- Added description to lvl 25 talents

**Comment by Wujekklawy** [comment]
@KaoticKanine yep, you're correct about **W**, and yea, i also think he's pretty cool support, he have "some" gr8 stuff but farming is not the best things he can do, Greaves will be most likly a must have on him, and actualy they work pretty well with his kit. Sorry it took me so long to respond
